### PR TITLE
Feature/geode 3977

### PR DESCRIPTION
**Pull request by mhansonp**
- Removed template arguments from QueryService
- Made some changes to pick up the nitpicks from the initial review.

### CODE REVIEW

File: clicache/src/QueryService.hpp [hunk 1]
```diff
@@ -22,6 +22,15 @@
...
 #include <geode/QueryService.hpp>
 #include "end_native.hpp"
+#include "Query.hpp"
+#include "Log.hpp"
+#include "CqAttributes.hpp"
+#include "CqQuery.hpp"
+#include "CqServiceStatistics.hpp"
+#include "impl/ManagedString.hpp"
+#include "ExceptionTypes.hpp"
+#include "impl/SafeConvert.hpp"
```
Comment (dgkimura) [inline clicache/src/QueryService.hpp]: Why are all these headers now needed?

File: clicache/src/QueryService.hpp [hunk 2]
```diff
@@ -51,80 +60,80 @@ namespace Apache
...
-        //generic<class TKey, class TResult>
+        generic<class TKey, class TResult>
         CqQuery<TKey, TResult>^ NewCq( String^ name, String^ query, CqAttributes<TKey, TResult>^ cqAttr, bool isDurable );
         /// @nativeclient
         /// <summary>
         /// Close all  <c>CqQuery</c> on this client.
         /// </summary>
         /// @endnativeclient
-	void CloseCqs();
+	      void CloseCqs();
```
Comment (dgkimura) [inline clicache/src/QueryService.hpp]: Nit - spacing seems a little off.  Mixed tabs/spaces. Same as some of other methods below.

File: clicache/src/QueryService.hpp (hunk 1 above)
Comment (mhansonp) [reply clicache/src/QueryService.hpp]: I *think they are, but I pulled their equivalents from the .cpp file, which they are not needed in.

File: clicache/src/QueryService.hpp (hunk 2 above)
Comment (mhansonp) [reply clicache/src/QueryService.hpp]: Went through and fixed the spacing...

**Pull request merged**